keys/constants.go

```diff
@@ -106,6 +106,9 @@ var (
 	LocalLeaseAppliedIndexSuffix = []byte("rlla")
 	// localRangeStatsSuffix is the suffix for range statistics.
 	LocalRangeStatsSuffix = []byte("stat")
+	// LocalTxnSpanThresholdSuffix is the suffix for the last txn span GC's
```

Comment:
From the comment, this should probably be `LocalTxnSpanGCThresholdSuffix`. Doesn't feel like there's otherwise enough context in the name.